playbooks/roles/l2tp-ipsec/tasks/main.yml

```diff
@@ -147,5 +145,15 @@
 # Ensure l2tp firewall rules are in place
 - include: firewall.yml
 
+- name: Enable and start ipsec and l2tp services
+  systemd:
+    daemon_reload: yes
+    name: "{{ item }}"
+    enabled: yes
+    state: restarted
+  with_items:
+    - ipsec
```

Comment:
Is there an existing `ipsec` service installed from upstream that this is using? 

I'm largely 100% ignorant about the IPsec/l2tp configuration Streisand uses but it seems like the former firewall init script called `ipsec setup start` and I'm wondering if that did something more than reloading the `ipsec` service will do.

@alimakki @jlund Do you know anything about this aspect?

Reply:
Right, `ipsec.service` comes from upstream. If you trace the execution of `ipsec setup start`, you'll see that all it does is couple of safe-checks and calling `systemctl start ipsec`. Moreover `ipsec.service` includes the same safe-checks too.

Reply:
@z0rc Aha, that makes sense. Thanks for clearing that up!